### PR TITLE
Content Review - QLD Resource Permit Status

### PR DESCRIPTION
**Pull request by KellyVance**
@DavidCrosswellGSQ GraphDB updated. Please publish to VocPrez Test and provide link

Custodian - @mark-gordon-dnrme 
Peer Reviewer 1 - @LizDerrington

### CODE REVIEW

**Comment by johnmckellar** [comment]
Vance & Andrew,
Thank you for consolidating the vocab, but I am a little concerned about the alt labels in:

gmps:application a skos:Concept ;
    skos:AltLabel "Not Accepted"@en,
        "Pre-Lodgement"@en,
        "Submitted"@en ;
    skos:definition "An application has been made for a permit."@en ;
    skos:inScheme gmps:conceptScheme ;
    skos:prefLabel "Application"@en ;
    skos:topConceptOf gmps:conceptScheme .

Cheers,
John

**Comment by KellyVance** [comment]
@johnmckellar 
We still want to be able to group some of these problem terms into one of the new and improved terms for internal retrospective purposes. But we do have to discourage their use and hide them from outward facing presentations. Therefore, I've reclassified Pre-lodgement and Not-accepted from skos:altLabel to skos:hiddenLabel properties.

File: vocabularies/qld-resource-permit-status.ttl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        gmps:abandoned,
+        gmps:application,
+        gmps:granted,
+        gmps:non-current,
+        gmps:unsuccessful ;
+    skos:prefLabel "Queensland Resource Permit Status"@en .
+
+gmps:abandoned a skos:Concept ;
+    skos:altLabel "Discarded"@en ;
+    skos:definition "An application for the permit that has been abandoned or discarded by the applicant."@en ;
```
Comment (LizDerrington) [inline vocabularies/qld-resource-permit-status.ttl]: 'An application for a permit that has been abandoned or discarded by the applicant'

File: vocabularies/qld-resource-permit-status.ttl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:non-current a skos:Concept ;
+    skos:definition "A permit that was once granted but is now no longer active."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Non-Current"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:unsuccessful a skos:Concept ;
+    skos:definition "An application for the permit was deemed unsuccessful by the Department."@en ;
```
Comment (LizDerrington) [inline vocabularies/qld-resource-permit-status.ttl]: "An application for a permit was deemed unsuccessful by the Department."

**Comment by KellyVance** [comment]
@mark-gordon-dnrme @LizDerrington 
Please review when you are able.

**Pull request merged**